test.js

```diff
@@ -5,8 +5,10 @@ const fs = sandboxedFs.bind('./test');
 
 fs.readFile('file.ext', (err, data) => {
   if (err) return console.log('Cannot read file');
+  console.log(data.toString());
 });
 
 fs.readFile('../../file.ext', (err, data) => {
   if (err) return console.log('Cannot read file');
```

Comment:
If you don't object to it, yes. I'd prefer to have `console.error` when errors are printed :)